internal/kubernetes/test_client.go

```diff
@@ -35,3 +36,26 @@ func (c *TestClient) CreatePersistentVolumeClaim(ctxt context.Context, pipeline
 	}
 	return pipeline, nil
 }
+
+func (c *TestClient) GetConfigMap(ctxt context.Context, cmName string, options metav1.GetOptions) (*corev1.ConfigMap, error) {
+	for _, cm := range c.CMs {
+		if cm.Name == cmName {
+			return cm, nil
+		}
+	}
+	return nil, fmt.Errorf("configmap %s not found", cmName)
```

Comment:
Please return an error here that can be tested if it is a "not found" error - see https://github.com/opendevstack/ods-pipeline/pull/421.